### ISSUE TITLE
Les pages les plus visitées

### ISSUE THREAD

**Issue by GILBM** [yvandouenel/pfca#2]
Dans les pages les plus visitées : 
La page création d'entreprise à Lunel, ne marche pas. Ça marque "un choix interdit a été détecté, veuillez contacter l'administrateur du site"
La page création d'activité à Lunel, idem même problème
La page création d'entreprise à Sète, idem même problème
La page BGE de l'Hérault, cela ne sort pas du tout la liste des BGE

**Comment by yvandouenel** [comment]
Oui, c'est un bug que l'on a déjà plusieurs fois signalé. Le fait de prendre la main sur l'outil va me permettre de vraiment corriger cela. J'ai commencé à démonter le mécanisme et je vois où ça bloque. Il me faudra 2 ou 3h pour changer en profondeur cela.
